apidoc/Titanium/Android/NotificationManager/NotificationManager.yml

```diff
@@ -40,6 +37,16 @@ methods:
         summary: Notification to display.
         type: Titanium.Android.Notification
 
+  - name: createNotificationChannel
+    osver: {android: {min: "8.0"}}
+    summary: Create a notification channel.
+    parameters:
+      - name: parameters
+        summary: Properties to set on a new object, including any defined by <Titanium.Android.NotificationChannel> except those marked not-creation or read-only.
```

Comment:
Document that this function will return null on Android OS version older than Android 8.